qstash/workflows/basics/api.mdx

```diff
@@ -0,0 +1,119 @@
+---
+title: "API"
+---
+
+# context.run
+
+`context.run` requires a step name and the code of the step as an async function. Each step can run for the full duration allowed by the platform. The result of each step is stored and remembered by QStash, allowing it to continue from where it left off in case of failure (e.g., endpoint downtime).
+
+Here are a couple of examples to demonstrate its usage:
```

Comment:
Some questions from the user’s POV:

What happens if my async function throws an exception?
Do I need to handle it manually, or will QStash automatically retry the function?
If it retries automatically, how can I configure the retry behavior; e.g. max number of retries?

Reply:
retry amount can't be configured at this moment. I was thinking of adding it in a later release.

I can continue working on it if you think it's a must.

Reply:
I think it's OK to postpone it for now.

My point is to put the answers in the documentation, even if we don’t support them.

Reply:
I will mention that the default is 3 and configuration will come soon in the doc.